manifests/init.pp

```diff
@@ -61,30 +66,25 @@
   validate_re($destination_directory, '.+')
   validate_re($filename, '.+')
 
-  if defined(File['C:\temp']) {
-    $tmp_dir = 'C:\temp'
-  } elsif defined(File['C:/temp']) {
-    $tmp_dir = 'C:/temp'
-  } else {
-    $tmp_dir = 'C:\temp'
-
-    file { $tmp_dir:
+  if ! defined(File['C:\temp']) {
+    file { 'C:\temp':
```

Comment:
Why are you changing this?

I feel that this hurts the interoperability of this module a bit.